sov-modules/sov-modules-api/src/default_context.rs

```diff
@@ -0,0 +1,71 @@
+use crate::default_signature::{DefaultPublicKey, DefaultSignature};
+use crate::{Address, AddressTrait, Context, PublicKey, Spec};
+
+use jmt::SimpleHasher;
+#[cfg(feature = "native")]
+use serde::{Deserialize, Serialize};
+use sov_state::DefaultStorageSpec;
+#[cfg(feature = "native")]
+use sov_state::ProverStorage;
+use sov_state::ZkStorage;
+use sovereign_sdk::core::types::ArrayWitness;
+
+// TODO: consider feature gating the serde implementations, since they are only needed for RPC
+// https://github.com/Sovereign-Labs/sovereign/issues/175
```

Comment:
I believe this comment is not relevant here, because whole `DefaultContext` is not available in zk (non-native) feature